linux_os/guide/system/network/network-iptables/iptables_ruleset_modifications/set_iptables_outbound_n_established_connections_configured/rule.yml

```diff
@@ -0,0 +1,47 @@
+documentation_complete: true
+
+title: 'Ensure outbound and established connections are configured'
+
```

Comment:
Please, also include `prodtype` restricting this rule to `sle` products. Other distributions can adopt this rule after if necessary.